docs/paypal.md

```diff
@@ -0,0 +1,169 @@
+# Pay with PayPal
+
+Accept PayPal payments in your Android app using the PayPal Payments SDK.
+
+## How it works
+
+This diagram shows how your client, your server, and PayPal interact:
+
+// TODO - Get a diagram of the payment flow similar to [this](https://developer.paypal.com/braintree/docs/start/overview#how-it-works)
+
+## Eligibility
+
+PayPal is available as a payment method to merchants in multiple [countries and currencies](https://developer.paypal.com/docs/checkout/payment-methods/).
+
+This SDK supports a minimum Android API of 23 or higher.
+Android apps can be written in Kotlin or Java 8 or higher.
+
+## How to integrate
+
+- [Custom Integration](#technical-steps---custom-integration)
+
+### Know before you code
+
+You will need to set up authorization to use the PayPal Payments SDK. 
+Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID and generate an access token. 
```

Comment:
>  and generate an access token

We can omit this part. We shouldn't need an access token to invoke the PayPal flow.

Reply:
What should the authorization for the order creation call be for this instead? (line 92)

Reply:
Ah, I was reading the doc with the lens of strictly the Android dev integrating the SDK. Since we are mentioning the create capture/authorize calls, keeping in the access token here makes sense.